Visual_Analysis.ipynb

```diff
@@ -122,9 +177,27 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "img_selection = 'logs/sample-model/pictures/*.png'\n",
```

Comment:
This breaks the out of the box 'demo'; can we do this in a different way?

Reply:
updated to set example as separate block and conditional running of S3

Reply:
Tested the new setup and it works for samples if you don't run the S3 section

Reply:
![image](https://github.com/aws-deepracer-community/deepracer-analysis/assets/53598199/2b1b6a42-ab2d-4a9a-82f5-7c16e5ab2bce)